pkg/agent/agent.go

```diff
@@ -26,16 +27,20 @@ type Klocksmith struct {
 	nc   v1core.NodeInterface
 	ue   *updateengine.Client
 	lc   *login1.Conn
+	ri   time.Duration
+	rt   time.Duration
 }
 
+const reapInterval = 10 * time.Second
```

Comment:
Let's have this be defaultInterval or something more general to replace the other occurrences of 10 second intervals